pkg/daemon/ceph/osd/daemon.go

```diff
@@ -353,6 +353,19 @@ func getAvailableDevices(context *clusterd.Context, desiredDevices []DesiredDevi
 						continue
 					}
 					logger.Infof("device %s matches device filter %s: %t", device.Name, desiredDevice.Name, matched)
+				} else if desiredDevice.IsDevlinksFilter {
+					for _, devlink := range strings.Fields(device.DevLinks) {
+						matched, err = regexp.Match(desiredDevice.Name, []byte(devlink))
+						if err != nil {
+							logger.Errorf("regex failed on device %s and filter %s. %+v", device.Name, desiredDevice.Name, err)
+							continue
+						}
+
+						if matched {
+							break
+						}
+					}
+					logger.Infof("device %s matches device filter %s: %t", device.Name, desiredDevice.Name, matched)
```

Comment:
```suggestion
					logger.Infof("device %q matches device filter %q", device.Name, desiredDevice.Name)
```

I don't think we need to show `matched`, or else it should be a degug log.